projects.yml

```diff
@@ -4416,6 +4416,12 @@ clj_new:
   url: https://github.com/seancorfield/clj-new
   categories: [tools.deps]
   platforms: [clj]
+  
+practicalli-clojure-deps.edn:
```

Comment:
Can you use underscores in the name, instead of dashes?

Reply:
Yes of course, apologies for the error.